app/frontend/src/cljs/teet/asset/assets_view.cljs

```diff
@@ -441,18 +435,78 @@
+       [:<>
+        [:div {:style {:position :fixed
+                       :left "0px" :top (common-styles/content-start)
+                       :width "100vw"
+                       :height "50px"
+                       :background-color theme-colors/gray-lighter
+                       :display :flex
+                       :flex-direction :row
+                       :align-items :center
+                       :justify-content :space-between}}
+         (tr [:asset :manager :result-count]
+             {:count (result-count-label results)})
+         [:div
+          [IconButton {:on-click toggle-result-mode!
+                       :color (if (= @mode :map)
+                                "primary"
+                                "secondary")}
+           [icons/maps-map]]
+          [IconButton {:on-click (r/partial set-mode! :filters)}
+           [icons/action-search]]]]
+        [:div {:style {:position :fixed
+                       :top (common-styles/content-start "50px")
+                       :height (common-styles/content-height "50px")
+                       :overflow-y :scroll}}
+         (if (= @mode :map)
+           [results-map {:e! e! :atl atl :criteria criteria
+                         :results results}]
+           (if-let [details (get-in app [:query :details])]
+             [context/consume :rotl
+              [result-details-view e! details atl]]
+             [results-table {:e! e! :atl atl :results results}]))]])]))
+
+
+(defn- desktop-view [e! {atl :asset-type-library :as app}
+                     {:keys [criteria query results]}]
   (r/with-let [filters-collapsed? (r/atom false)]
-    (if-not atl
-      [CircularProgress]
-      [context/provide :rotl (asset-type-library/rotl-map atl)
-       [vertical-split-pane {:minSize 50
-                             :defaultSize (if @filters-collapsed? 30 330)
-                             :allowResize false}
-        [asset-filters e! atl criteria filters-collapsed?]
-        [context/provide :rotl (asset-type-library/rotl-map atl)
-         [assets-results {:e! e! :atl atl :criteria criteria
-                          :asset-query query
-                          :results results
-                          :details (get-in app [:query :details])}]]]])))
+    [vertical-split-pane {:minSize 50
+                          :defaultSize (if @filters-collapsed? 30 330)
+                          :allowResize false}
+     [asset-filters-container e! atl criteria filters-collapsed?]
+     [context/provide :rotl (asset-type-library/rotl-map atl)
+      [assets-results {:e! e! :atl atl :criteria criteria
+                       :asset-query query
+                       :results results
+                       :details (get-in app [:query :details])}]]]))
+
+(defn assets-page [e! {atl :asset-type-library :as app} state]
+  (if-not atl
+    [CircularProgress]
+    [context/provide :rotl (asset-type-library/rotl-map atl)
+     (if (responsivity-styles/mobile?)
```

Comment:
maybe instead of this we could use css breakponts to display none either of these? The onresize listener feels a bit inconsistent sometimes.

Reply:
I don't want to render the heavy map/listing components twice, but it could perhaps be refactored